webscalesqlclient/src

```diff
@@ -1 +1 @@
-webscalesql-5.6/
\ No newline at end of file
+../../build/third-party/webscalesqlclient/webscalesql-5.6
```

Comment:
If you want to fix out-of-tree builds in a separate PR, we can discuss there. But this is quite broken-as is. It's hard-coding a different path, one which most people don't use :)